### DISCUSSION TITLE
2005/12/12/%E9%9D%A2%E8%AF%95%E5%AE%98%E7%9A%84%E9%9D%A2%E7%BB%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#69] [Announcements]
# 2005/12/12/%E9%9D%A2%E8%AF%95%E5%AE%98%E7%9A%84%E9%9D%A2%E7%BB%8F

阮奇桢现在的毕业生，在参加用人单位面试之前，都要上网查一下往届学生以及先面试者写的面经。我不知道面试官们是否也会上网查一下有没有专供面试官使用的面经。

https://blog.qizhen.xyz/2005/12/12/%E9%9D%A2%E8%AF%95%E5%AE%98%E7%9A%84%E9%9D%A2%E7%BB%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Carrie
2007年05月29日 18:29 编辑
写得真好。对我很受益。
刚从事HR的工作，做的就是招聘，感觉有点“疲惫”，也许因为没有前辈的指导。
偶然间看到楼主的文章，顿觉自己在无形中错失了很多东西。看来，我要好好补补课了。
谢谢你！

回复
nettie
2007年05月29日 14:03 编辑
6月毕业的我也面试几次,感觉还好…有些考官狐假虎威…不过大多数是好人..

回复
cynthia-lin
2007年05月29日 10:06 编辑
谢谢楼主，辛苦了，说起来，我一共也就面试了2次，这方面确实经验太少

回复
CHIU JAN
2007年05月29日 08:06 编辑
我工作面试还早呢~不过可以有其他的面试啊

回复
w
2007年05月28日 14:06 编辑
应该是同行吧，有时间要多向您请教，先谢谢了

回复
晓栋
2007年05月28日 08:10 编辑
英语口语不好 也能进美国外企 楼主真是很有实力的一个人

回复
少华
2007年05月26日 21:31 编辑
虽然没什么实际参考价值，但是第一次看到这么详细的面试官的自我分析解剖还是挺有趣的，呵呵~~

回复
clover
2007年05月26日 00:26 编辑
唔….越看越觉得自己没信心去面试了…..

回复
Aiqin
2007年05月25日 14:24 编辑
好长啊…..
 

回复
Sissi
2007年05月25日 08:15 编辑
写这么多，花了多久阿！辛苦了！

回复
等待风停
2007年05月22日 13:47 编辑
很好 谢谢

回复
CHiN
2007年05月20日 14:45 编辑
有点长~下来仔细看看
欢迎回访

回复
飞
2007年05月19日 21:53 编辑
好长~

回复
芳
2007年05月17日 08:37 编辑
受益良多!

回复
Unknown
2007年05月12日 23:31 编辑
恩 有用！ —掘客

回复
欣瑢
2007年05月10日 16:17 编辑
厉害

回复
冰水为之
2007年05月9日 11:28 编辑
麻烦事麻烦，要百折不挠，勇往直前 。共同加油。

回复
静
2007年05月6日 21:31 编辑
做了被面试者好几年，看完以后觉得受益颇多～

回复
黎
2007年05月6日 15:13 编辑
我就面霸，次次下来总结反省，希望早日摆脱这两个字^O^

回复
Exen
2007年05月4日 13:27 编辑
这就是网络的好处，在生活中断然是没有人会跟你讲这些的。我们应该怀着一颗感激的心去读，去评论。感谢原作者啊！

回复
羽落
2007年05月3日 17:29 编辑
中午时候,已经看过了.写的非常好.
很理性,很客观,很有建设性
虽然看到某些细节触动我内心的隐痛
会经常来看看

回复
麟
2007年04月29日 09:49 编辑
先占个地方

回复
假扮の公主
2007年04月27日 14:05 编辑
总算看完了。。好长的经验吖。。不过很有价值感谢与我们分享啊~~~~~

回复
summer
2007年04月27日 01:12 编辑
实在是太……厉害了。。

回复
花
2007年04月26日 19:13 编辑
好棒哦～

回复
090
2007年04月25日 22:47 编辑
这么有见解，哈哈……

回复
gomer
2007年04月24日 14:29 编辑
我是来拉人的，来吧，来吧，吧唧吧唧飞一个

回复
♀安娜泉♀
2007年04月23日 18:49 编辑
真的长见识了，呵呵，但愿一年后的面试能顺利通过

回复
Vivien
2007年04月23日 14:54 编辑
从首页路过,前辈你以后适合做讲师~

回复
*moon*
2007年04月22日 13:06 编辑
学习!

回复
arleie
2007年04月21日 16:24 编辑
  学习学习咯
      为日后找工作打伏笔吧
生活就这样

回复
fish
2007年04月19日 13:53 编辑
第一次去面试也是笨笨的感觉。。呵呵，不过看看别人的也是一样超可爱

回复
Yarkee
2007年04月18日 10:23 编辑
谢谢你的日志哦
学到好多东西呢
 

回复
wen
2007年04月17日 16:39 编辑
找工作中，来学习经验，最重要的还是要活学活用哈。

回复
af27
2007年04月17日 14:11 编辑
嗯，中午，困

回复
leehom
2007年04月11日 12:02 编辑
您好，我是SPACES版主。您的日志已被风云榜收录。http://msnpopularspaces.spaces.live.com/请及时关注SPACES首页。http://spaces.live.com/?mkt=zh-cn相关栏目投稿请进风云榜首页：http://msnpopularspaces.spaces.live.com/

回复
Bei
2006年10月14日 00:56 编辑
Google了一下“面经”，就连到这里来了，从没面试过，看了你的美文还是觉得受益匪浅的，谢谢拉

回复
Ansley
2006年01月13日 22:45 编辑
挺有耐心,写了这么长.大白呼的潜能体现出来了:-)

回复
qingxi
2005年12月24日 17:20 编辑
有帮助哟。

回复
Magnus
2005年12月24日 12:48 编辑
好文章，我现在正要找工作呢

回复
Qizhen
2005年12月22日 22:00 编辑
年底了，在忙着工作啊 ^_^*今天再稍写一点 :)

回复
幽
2005年12月22日 05:02 编辑
顶！！！马上要开始找工作了！ 加油写哦！ 我期待ING ，对我来讲是个很大帮助呢！ “` 对了，你好强哦！佩服！ ：）

回复
夕颜
2005年12月21日 10:16 编辑
这样的老师就不对了,上课上一半就不来了，我还等着下面的内容呢!

回复
duoduo
2005年12月16日 16:59 编辑
面考和被面试看来真是一门艺术的2个分支！

回复
夕颜
2005年12月15日 13:19 编辑
呵呵,还没写完哦,那你的经验还真不少,建议你用个滚动框,没有横条的那种,这样的你空间就更完美了,不嫌弃可以去我那边的教室拿

回复
Su━╃﹎ˊ｛釹魜｝™丶╄━½
2005年12月15日 12:27 编辑
受益匪淺….期待下文中….

回复
zhefei
2005年12月15日 10:02 编辑
长见识中……

回复
奕瑋
2005年12月13日 10:30 编辑
说实话，真是越来越欣赏阮主了。

回复
竹贤
2005年12月13日 08:26 编辑
请继续。

回复
Jing
2005年12月12日 22:32 编辑
:-)受教呢！
